infra/README.md

```diff
@@ -0,0 +1,65 @@
+# Meshdb Environment Setup
+
+These instructions will set up a 4 node k3s cluster on proxmox.
+- 1 "manager" node for control plane and to be used for deployments.
+- 3 "agent" nodes to run services.
+
+1. Configure a user for the [proxmox provider](https://registry.terraform.io/providers/Telmate/proxmox/latest/docs) and set up env vars. Create an API key in Proxmox, and disable Privilege Separation.
+2. Setup tfvars + ssh keys
+```
+cd meshdb/infra/tf/
+cp example.tfvars your_env.tfvars
+# Modify your_env.tfvars to meet your needs
+ENV_NAME="garfield" # Matching meshdb_env_name
+ssh-keygen -t ed25519 -f ./meshdb$ENV_NAME
+```
+3. Create the VMs that will host k3s
+```
+terraform init
+terraform plan --var-file=your_env.tfvars
+terraform apply --var-file=your_env.tfvars
+```
+4. Login via serial and figure out the IPs that were received from DHCP
+5. One time provisioning for the manager node
+
+```
+target_host="<MGR IP>"
+scp infra/mgr_provision.sh ubuntu@$target_host:/home/ubuntu/mgr_provision.sh
+ssh -t ubuntu@$target_host "sudo bash /home/ubuntu/mgr_provision.sh"
```

Comment:
`ubuntu` is the default here, but since it's a variable, it could be changed. I wonder if this could be accounted for in the scripts.

Reply:
Also, I needed to provide the `-i` argument with the private key we made.

Reply:
Consolidated step